core/src/test/java/org/arquillian/smart/testing/impl/TestStrategyApplierUsingPropertyTest.java

```diff
@@ -135,4 +140,9 @@ private TestExecutionPlannerLoader prepareLoader(final Set<Class<?>> testsToRun,
     private Set<Class<?>> createTestsToRun(Class... testClasses) {
         return new LinkedHashSet<>(Arrays.<Class<?>>asList(testClasses));
     }
+
+    private String getBaseDir() {
```

Comment:
you told me:

> yes we need this, otherwise we have to hard code this value to "core" to work it from IDEA & terminal.

I tried it with these options:
* `System.getProperty("user.dir")`
* `new File(".").getAbsolutePath()`
* removing the `in(getBaseDir())` completely

and in all cases the tests are green on both command line and IDE

Reply:
I know why it's passing because you added fix for this few days ago in #181